pkg/agent/prommetrics.go

```diff
@@ -30,13 +44,49 @@ func makePrometheusParts(globalstate *agentState) (PromMetrics, *prometheus.Regi
 	metrics := PromMetrics{
 		// the util.RegisterMetric() function registers the collector and returns
 		// it so we can set it directly on the output structure.
+
+		// ---- SCHEDULER ----
 		schedulerRequests: util.RegisterMetric(reg, prometheus.NewCounterVec(
 			prometheus.CounterOpts{
 				Name: "autoscaling_agent_scheduler_plugin_requests_total",
 				Help: "Number of attempted HTTP requests to the scheduler plugin by autoscaler-agents",
 			},
 			[]string{"code"},
 		)),
+		schedulerRequestedChange: resourceChangePair{
+			cpu: util.RegisterMetric(reg, prometheus.NewCounterVec(
+				prometheus.CounterOpts{
+					Name: "autoscaling_agent_scheduler_plugin_requested_cpu_change_total",
+					Help: "Total change in CPU requested from the scheduler",
+				},
+				[]string{"direction"},
```

Comment:
maybe direction should be a constant

Reply:
Makes sense. Made "direction" and "up" and "down" all constants.